storage/sql/crud.go

```diff
@@ -140,10 +140,16 @@ func (c *conn) UpdateAuthRequest(id string, updater func(a storage.AuthRequest)
 			return err
 		}
 
+		err = c.flavor.lockForUpdate(tx, "auth_request", "id", r.ID)
+		if err != nil {
+			return fmt.Errorf("update auth request: %v", err)
```

Comment:
💭 All these `fmt.Errorf` make me nervous -- they're hard to deal with in call sites -- but this is the style this package is written in, so it's fine at add a few. 👍

Reply:
We can switch to github.com/pkg/errors in another PR :)